tests/integration/test_bundle.py

```diff
@@ -8,9 +8,14 @@
 
 from pytest_operator.plugin import OpsTest
 
+from tests.integration.constants import MYSQL_APP, ROUTER_APP, TLS_APP
+
 logger = logging.getLogger(__name__)
 
 
-async def test_placeholder(ops_test: OpsTest) -> None:
-    """A placeholder for an integration test to be implemented."""
-    assert True
+async def test_deploy_bundle(ops_test: OpsTest) -> None:
+    """Deploy bundle."""
+    async with ops_test.fast_forward():
+        await ops_test.model.deploy("./releases/latest/mysql-k8s-bundle.yaml", trust=True)
+
+        await ops_test.model.wait_for_idle(apps=[MYSQL_APP, ROUTER_APP, TLS_APP], timeout=5 * 60)
```

Comment:
should we assert that the apps reach a certain status? (even if it is `waiting`)